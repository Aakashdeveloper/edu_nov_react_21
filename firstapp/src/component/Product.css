.mainDiv{
    margin-left: 10%;
    margin-right: 10%;
}
.card{
    height:300px;
    width:20%;
    margin-left: 5%;
    margin-top: 2%;
    display:inline-block;
    box-shadow: 3px 5px 5px gray;
}