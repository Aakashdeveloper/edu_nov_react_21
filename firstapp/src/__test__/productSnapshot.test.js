import React from 'react';
import Product from '../component/ProductDisplay';
import {create} from 'react-test-renderer';

let data = [
    {
        "id":1,
        "brand":"3M",
        "discount":"Less than 10%",
        "form": "Mask",
        "use":"Respiratory Care",
        "name":"3M 9004IN Particle Respirator Mask",
        "image":"https://i.ibb.co/8BJhgK2/r6xkegcloxgn5eq8owwg.jpg",
        "description":"box of 10 masks",
        "cost":230
    },
    {
        "id":2,
        "brand":"Vogmask",
        "discount":"10% and above",
        "form": "Mask",
        "uses":"Respiratory Care",
        "name":"Vogmask Vector Havoc N99CV Mask Large",
        "image":"https://i.ibb.co/fHQ8JY4/kclatp7d9dt0dwgu4zkq.jpg",
        "description":"box of 10 masks",
        "cost":300
    }
];

describe('Snapshot Testing',() => {
    test('Testing Product',() => {
        let tree = create(<Product prodData={data}/>)
        expect(tree.toJSON()).toMatchSnapshot()
    })
});