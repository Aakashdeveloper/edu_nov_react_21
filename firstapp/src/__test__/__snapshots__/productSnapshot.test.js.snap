// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Testing Testing Product 1`] = `
<div
  className="mainDiv"
>
  <div
    className="card"
  >
    <img
      src="https://i.ibb.co/8BJhgK2/r6xkegcloxgn5eq8owwg.jpg"
      style={
        Object {
          "height": 100,
          "width": 100,
        }
      }
    />
    <h2>
      3M 9004IN Particle Respirator Mask
    </h2>
    <p>
      3M
    </p>
  </div>
  <div
    className="card"
  >
    <img
      src="https://i.ibb.co/fHQ8JY4/kclatp7d9dt0dwgu4zkq.jpg"
      style={
        Object {
          "height": 100,
          "width": 100,
        }
      }
    />
    <h2>
      Vogmask Vector Havoc N99CV Mask Large
    </h2>
    <p>
      Vogmask
    </p>
  </div>
</div>
`;
